src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -210,23 +208,8 @@ public partial class Socket : System.IDisposable
         public void Connect(string host, int port) { }
         public bool ConnectAsync(System.Net.Sockets.SocketAsyncEventArgs e) { return default(bool); }
         public static bool ConnectAsync(System.Net.Sockets.SocketType socketType, System.Net.Sockets.ProtocolType protocolType, System.Net.Sockets.SocketAsyncEventArgs e) { return default(bool); }
-        public void Disconnect(bool reuseSocket) { }
```

Comment:
@josguil If we remove `Disconnect` and `DisconnectAsync` we'll need to also remove `SocketAsyncEventArgs.DisconnectReuseSocket`.